ZAPDTR/ZAPDUtils/Utils/Directory.h

```diff
@@ -4,6 +4,10 @@
 #include <string>
 #include <vector>
 
+#ifdef __SWITCH__
+long int pathconf (const char *__path, int __name){}
```

Comment:
Not a fan of this approach. This can quickly lead to multiple definitions when included in multiple files.
The Wii U port places this in a seperate file: <https://github.com/GaryOderNichts/Shipwright/blob/wiiu/soh/wiiu/pathconf.c>